examples/point-cloud-ply/utils/orbit-control.js

```diff
@@ -0,0 +1,177 @@
+/* global window */
```

Comment:
yes, two separate examples. we talked about to consolidate all the viewport + interaction logics, after that I can just import the orbit controller.